Misc/NEWS.d/next/Library/2020-05-02-01-01-30.bpo-29753.n2M-AF.rst

```diff
@@ -0,0 +1,3 @@
+Fixes packed bitfields not being calculated properly in some cases (see
+bpo-29753).
```

Comment:
Please do not mention the issue number in the news entry (it is automatically added).